### PR TITLE
Publish Java client through Jitpack

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
## Change list

Publish Java client through Jitpack
 
## Types of changes

What types of changes are you proposing/introducing to Java client?
_Put an `x` in the boxes that apply_

- [ ] No changes in production code.
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

@mykola-mokhnach @jlipps This PR will allow us to publish java client through jitpack. This will unblock users to use latest commit even before its available in maven central.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -10,7 +10,7 @@ apply plugin: 'maven-publish'
 import org.apache.tools.ant.filters.*
 
 group 'io.appium'
-version '6.1.0'
+version '6.1.1-SNAPSHOT'
```
Comment (mykola-mokhnach) [inline build.gradle]: don't we call it BETA1?

Comment (SrinivasanTarget) [reply build.gradle]: Ideally we used to call as 'SNAPSHOT' in maven world before we release either beta or major version.

File: README.md [hunk 2]
```diff
@@ -17,6 +17,54 @@ This is the Java language binding for writing Appium Tests, conforms to [Mobile
...
+			maven { url 'https://jitpack.io' }
+		}
+	}
+```
+
+ - Add the dependency:
+ 
+```
+    dependencies {
+		implementation 'com.github.User:Repo:Tag'
```
Comment (mykola-mokhnach) [inline README.md]: can we set the data for java client in this example?

Comment (SrinivasanTarget) [reply README.md]: Sure will update

**Comment by mykola-mokhnach** [comment]
I can observe errors in the logs under https://jitpack.io/#appium/java-client. Shouldn't these logs be green?

**Comment by saikrishna321** [comment]
yah it should be green, build is failing because it's not skipping the sign archives

**Comment by SrinivasanTarget** [comment]
@mykola-mokhnach It wont be green until unless it is merged or atleast should be a branch in upstream to run. Check [here](https://jitpack.io/#srinivasantarget/java-client/jitpublish-SNAPSHOT)

**Comment by SrinivasanTarget** [comment]
Here is the build [logs](https://jitpack.io/com/github/srinivasantarget/java-client/jitpublish-v6.1.0-g02eccdc-27/build.log) from jitpack

**Comment by mykola-mokhnach** [comment]
Ok, I believe you know better

**Pull request merged**